README.md

```diff
@@ -3326,6 +3326,7 @@ _Software written in Go._
 - [drive](https://github.com/odeke-em/drive) - Google Drive client for the commandline.
 - [Duplicacy](https://github.com/gilbertchen/duplicacy) - A cross-platform network and cloud backup tool based on the idea of lock-free deduplication.
 - [fjira](https://github.com/mk-5/fjira) - A fuzzy-search based terminal UI application for Attlasian Jira
+- [Flox](https://flox.dev) - An environment and package manager in one, useful for getting specific version of Go.
```

Comment:
This is not a go project !

This repository is intended to include projects that are written in Go.

Reply:
Ah! My apologies. I thought it'd also be okay if the project was capable of supporting Go projects.
Please excuse my misunderstanding. I'll close the PR. 😊